connectivity/netsocket/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# TODO CMake: Perhaps move this/these file(s)
```

Comment:
Where do you propose we move them to? Could you explain in the `TODO`?

Reply:
Address in #13870